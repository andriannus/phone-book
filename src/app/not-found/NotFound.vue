<template>
  <div class="NotFound">
    <p class="FontSize-2xlarge MarginBottom">Page Not Found</p>

    <router-link class="Button Button--primary" to="/landing">
      Take me away
    </router-link>
  </div>
</template>

<script>
export default {
  name: "NotFound",
};
</script>

<style lang="scss" scoped>
.NotFound {
  align-items: center;
  display: flex;
  flex-direction: column;
  justify-content: center;
  min-height: 100vh;
}
</style>
