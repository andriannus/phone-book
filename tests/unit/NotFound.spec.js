import { shallowMount } from "@vue/test-utils";

import NotFound from "@/app/not-found/NotFound.vue";

describe("NotFound.vue", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(NotFound);
  });

  it("should create", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
