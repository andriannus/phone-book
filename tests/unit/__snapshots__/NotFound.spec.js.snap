// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.vue should create 1`] = `
VueWrapper {
  "__app": Object {
    "_component": Object {
      "__emits": Object {},
      "__props": Array [
        Object {},
        Array [],
      ],
      "name": "VTU_ROOT",
      "render": [Function],
    },
    "_container": <div
      data-v-app=""
    >
      <div
        class="NotFound"
      >
        <p
          class="FontSize-2xlarge MarginBottom"
        >
          Page Not Found
        </p>
        <router-link
          class="Button Button--primary"
          to="/landing"
        >
           Take me away 
        </router-link>
      </div>
    </div>,
    "_context": Object {
      "app": [Circular],
      "components": Object {},
      "config": Object {
        "errorHandler": undefined,
        "globalProperties": Object {},
        "isCustomElement": [Function],
        "isNativeTag": [Function],
        "optionMergeStrategies": Object {},
        "performance": false,
        "warnHandler": undefined,
      },
      "directives": Object {},
      "mixins": Array [
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
        Object {
          "__emits": null,
          "__props": Array [],
          "beforeCreate": [Function],
        },
      ],
      "provides": Object {},
      "reload": [Function],
    },
    "_props": null,
    "_uid": 0,
    "component": [Function],
    "config": Object {
      "errorHandler": undefined,
      "globalProperties": Object {},
      "isCustomElement": [Function],
      "isNativeTag": [Function],
      "optionMergeStrategies": Object {},
      "performance": false,
      "warnHandler": undefined,
    },
    "directive": [Function],
    "mixin": [Function],
    "mount": [Function],
    "provide": [Function],
    "unmount": [Function],
    "use": [Function],
    "version": "3.0.2",
  },
  "__functionalEmits": Object {},
  "__setProps": [Function],
  "componentVM": Object {
    "__emitted": Object {},
  },
  "rootVM": Object {
    "__emitted": Object {},
  },
}
`;
